BRCrypto.c

```diff
@@ -556,31 +556,31 @@ void BRMD5(void *md16, const void *data, size_t len) {
 #define fmix32(h) ((h) ^= (h) >> 16, (h) *= 0x85ebca6b, (h) ^= (h) >> 13, (h) *= 0xc2b2ae35, (h) ^= (h) >> 16)
 
 // murmurHash3 (x86_32): https://code.google.com/p/smhasher/ - for non-cryptographic use only
-uint32_t BRMurmur3_32(const void *data, size_t len, uint32_t seed) {
+uint32_t BRMurmur3_32(const void *data, size_t dataLen, uint32_t seed)
+{
```

Comment:
Please respect our style going... the `{` comes into the same line as the function definition. I saw this commit on the `breadwallet` og fork.